website/pages/en/cookbook/arweave.mdx

```diff
@@ -83,7 +83,7 @@ dataSources:
 ```
 
 - Arweave subgraphs introduce a new kind of data source (`arweave`)
-- The network should correspond to a network on the hosting Graph Node. On the hosted service, Arweave's mainnet is `arweave-mainnet`
+- The network should correspond to a network on the hosting Graph Node. in Subgraph Studio, Arweave's mainnet is `arweave-mainnet`
```

Comment:
"in Subgraph Studio" should be capitalized.

Reply:
@MichaelMacaulay – When finding & replacing, "match case" is your friend. :P